### ISSUE TITLE
Project idea to promote R packages in populations genetics

### ISSUE THREAD

**Issue by smanel** [NESCent/r-popgen-hackathon#7]
Not fully clear of how and what but a suggestion below on how promoting R packages in population genetics, since it is probably one of the end-product of our working group (?):
 A way to do is to publish well –driven applications using (almost) only R packages (already existing and developed during the hackathon): it can help users to better know about and to see the possibilities.  This requires using free existing databases already published, and to address   hot topic questions in populations genetics (e.g. detection of climatic signal of selection, ....). It would be wonderful  to have unpublished databases, but probably not possible.

**Comment by DrK-Lo** [comment]
Would using R and "system()" as a front to call other programs written in C (Bayenv, LFMM, etc) count under this umbrella?

**Comment by darencard** [comment]
Having a wrapper for many popular stand-alone programs would be welcomed by many I think. No sense in re-inventing the wheel, though in some cases it may be necessary to reimplement things. It would be especially helpful to provide integration with the other capabilities (e.g., plotting) in R. I also think that the pseudo-GUI environment provided by RStudio and certain packages could bridge the gap to those with little experience with large genetic datasets.

**Comment by smanel** [comment]
Yes using R and "system()" as a front to call other programs (Bayenv, LFMM)  if possible  would be very useful...
The same for the detection of population genetic structure like STRUCTURE or other bayesian clustering approaches...

**Comment by thibautjombart** [comment]
I can see the appeal but this typically poses dependency problems and will
make a package trickier to get onto CRAN. The use the system() for system
calls is also probably fairly platform dependent.

**Comment by KlausVigo** [comment]
What would be the requisites for a population genetics task view or mailing list on CRAN, like the [task view](http://cran.r-project.org/web/views/Phylogenetics.html) and the [mailing list](https://stat.ethz.ch/mailman/listinfo/r-sig-phylo) for phylogentics and comparative methods? This would give pop gene users in R an overview over available packages and available functionality right on the R website and a source for FAQs.

**Comment by thibautjombart** [comment]
They exist already:
http://cran.r-project.org/web/views/Genetics.html

https://stat.ethz.ch/mailman/listinfo/r-sig-genetics

R-sig-genetics has been massively under-used so far - just a matter of
getting the right people in though.
